test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,30 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNotNull();
+
+        // only one object
+        assertAnyNull((Object) null);
+        assertNotNull(1);
+        assertNotNull("");
+        assertNotNull("abc");
+
+        // all objects not null
+        assertNotNull("abc", "ab", "a");
+        assertNotNull(1, 2);
+
+        // some null objects
+        assertAnyNull(null, "abc");
+        assertAnyNull("abc", "", "abc", null);
+        assertAnyNull("", "abc", null, "abc");
+        assertAnyNull(1, null);
+        assertAnyNull(null, 1, Integer.valueOf(1));
+        assertAnyNull(null, null);
+        assertAnyNull(null, "a", "b", null);
```

Comment:
Sometimes when adding test cases, think about how the method treat this new test case differently from the other existing ones.